pkg/container-disk/container-disk_test.go

```diff
@@ -181,7 +181,29 @@ var _ = Describe("ContainerDisk", func() {
 					},
 				}
 				containers := GenerateContainers(vmi, "libvirt-runtime", "/var/run/libvirt")
-				Expect(containers[0].Resources.Limits).To(HaveLen(2))
+
+				containerResourceSpecs := []k8sv1.ResourceList{containers[0].Resources.Limits, containers[0].Resources.Requests}
+
+				for _, containerResourceSpec := range containerResourceSpecs {
+					Expect(containerResourceSpec).To(And(HaveKey(k8sv1.ResourceCPU), HaveKey(k8sv1.ResourceMemory)))
+				}
+			})
+			It("by verifying that ephemeral storage request is set to every container", func() {
+
+				vmi := v1.NewMinimalVMI("fake-vmi")
+				appendContainerDisk(vmi, "r0")
+				containers := GenerateContainers(vmi, "libvirt-runtime", "/var/run/libvirt")
+
+				expectedEphemeralStorageRequest := resource.MustParse("50M")
```

Comment:
I agree, the question is where to defined this constant. 50M is needed both for compute container (which is defined in template.go) and for container disk (which is defined in container-disk.go).

I can define a different constant for each one since they don't have to be equal in their size.